handlers/movies.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"go_project/models"
+
+	"github.com/labstack/echo/v4"
+)
+
+func GetMovieById(c echo.Context) error {
+	id := c.Param("id")
+	movie, err := models.GetMovieById(id)
+	if err != nil {
+		response := map[string]string{"error": err.Error()}
+		return c.JSON(404, response)
```

Comment:
You can use the **_net/http_** package to avoid using magic numbers:
```go
return c.JSON(http.StatusNotFound, response)
```

Reply:
Added status codes!